tests/unit_tests/data_steward/utils/gcs_test.py

```diff
@@ -0,0 +1,80 @@
+# Python imports
+from io import BytesIO
+from unittest import TestCase
+from unittest.mock import patch, MagicMock
+from typing import Callable
+
+# Third party imports
+
+# Project imports
+from utils import gcs
+
+
+class GCSTest(TestCase):
```

Comment:
This test class should not be necessary as the implementors should utilize known-functional capabilities within the storage client directly.

Reply:
Post discussion notes:
1. Remove unused tests